guardrails/cli/hub/template.py

```diff
@@ -0,0 +1,24 @@
+import json
+import os
+
+from guardrails.cli.server.hub_client import get_guard_template
+
+
+def get_template(template_name: str) -> tuple[dict, str]:
+    # if template ends in .json load file from disk relative to the execution directory
+    if template_name.endswith(".json"):
+        template_file_name = template_name
+        file_path = os.path.join(os.getcwd(), template_name)
+        with open(file_path, "r") as fin:
+            return json.load(fin), template_file_name
```

Comment:
Are the system errors clear enough if the file doesn't exist?

Reply:
Also do we want to run a schema check on the json to make sure they specified it correctly?

Reply:
good catch let me add some better handlers